### PR TITLE
Editorial: Align with Web IDL specification

### PR DESCRIPTION
**Pull request by autokagami**
This is an automated pull request to align the spec with the latest Web IDL specification.

  Currently the autofix might introduce some awkward code formatting, so please feel free to modify the formatting.

  Please file an issue on https://github.com/saschanaz/webidl-updater/issues/new if you think this PR is invalid or should be enhanced.

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -228,6 +229,7 @@ but retransmissions may be disabled or the stream may aborted to produce a form
 of unreliability.  All stream data is encrypted and congestion-controlled.
 
 <pre class="idl">
+[Exposed=Window]
```
Comment (ricea) [inline index.bs]: This should be changed to "interface mixin" rather than exposed.

**Comment by saschanaz** [comment]
Or you can fix here?

**Comment by marcoscaceres** [comment]
Forgot I had magical powers to fix things 🧙‍♂️

**Comment by marcoscaceres** [comment]
@saschanaz weird... 

FATAL ERROR: IDL SYNTAX ERROR LINE: 2 - skipped: "Promise<SendStream> createSendStream(optional SendStreamParameters parameters = {})"
 ✘  Did not generate, due to fatal errors

**Comment by marcoscaceres** [comment]
Does the IDL parser used by BS not support `optional = {}` yet? I thought it did.

**Comment by saschanaz** [comment]
The bikeshed version 8b51bf63605cf469d25c80aa1498b4fb96c84530 seems quite old, maybe you should update.

**Comment by marcoscaceres** [comment]
how weird... I literally just did `bikeshed update` before generating this doc.

**Comment by marcoscaceres** [comment]
might need to reinstall? ... will try that

**Comment by marcoscaceres** [comment]
seems I had an old version installed from when I was hacking on Bikeshed... turns out Bikeshed update doesn't actually update Bikeshed? just the data maybe. Reinstalling it.

**Comment by saschanaz** [comment]
BTW you can just `pip install bikeshed` and update using pip 🎉

**Pull request merged**